runtime/src/iree/hal/drivers/vulkan/vulkan_headers.h

```diff
@@ -41,10 +41,10 @@
 #define VK_USE_PLATFORM_WIN32_KHR 1
 #endif  // IREE_PLATFORM_*
 
-#include <vulkan/vulkan.h>  // IWYU pragma: export
+#include "third_party/vulkan_headers/include/vulkan/vulkan.h"  // IWYU pragma: export
```

Comment:
Huh this looks a bit weird to me, though I'm not sure I actually have a good technical reason. I've always seen things rooted at the `include/` and not including it. I guess this is necessary to avoid accidentally getting someone else's Vulkan headers (including the system's)?